Formula/mercury.rb

```diff
@@ -16,20 +15,16 @@ class Mercury < Formula
     sha256 "0e736ef6f5cc48bc9d6f7d50cb9df6fb52dba2b0b3bf2d83b378f83fcff4ecb9" => :mavericks
   end
 
+  depends_on :java => "1.8"
```

Comment:
does it not work with the latest java release?

Reply:
Actually, it does. Should that be:
```
  depends_on :java => "1.8+"
```
instead?

Reply:
`depends_on "openjdk"`

Reply:
:ok_hand: